### PR TITLE
scrub ie9 related workarounds

### LINKED ISSUE #3243: Drop IE9 support; support IE11 and Edge only
Microsoft doesn't even support it anymore so neither should we

### PR DESCRIPTION
**Pull request by hellochar**
Fixes #3243 

changes work fine in ie11

### CODE REVIEW

File: src/utils/windowUtils.ts [hunk 1]
```diff
@@ -15,13 +15,7 @@ export function warn(warning: string) {
     return;
```
Comment (adidahiya) [inline src/utils/windowUtils.ts:15]: we should remove `Configs` and instead check `process.env.NODE_ENV !== "production"`

**Review by adidahiya — COMMENTED** [review]
nice

File: plottable.js [hunk 2]
```diff
@@ -15948,14 +15948,7 @@ function warn(warning) {
         return;
     }
     /* tslint:disable:no-console */
```
Comment (leebyp) [inline plottable.js]: `// tslint:disable-next-line:no-console`

File: test/utils/windowUtilsTests.ts [hunk 3]
```diff
@@ -1,51 +1,20 @@
...
-      it("uses console.warn if available, console.log otherwise", () => {
-        const originalWindowWarn = console.warn;
-        let receivedWarning: string = null;
-        const replacementWarn = (warning: string) => receivedWarning = warning;
-        console.warn = replacementWarn;
-
-        const expectedWarning = "BEEP BOOP";
-        Plottable.Utils.Window.warn(expectedWarning);
-        console.warn = originalWindowWarn;
-        assert.strictEqual(receivedWarning, expectedWarning, "console.warn was passed the expected message");
```
Comment (leebyp) [inline test/utils/windowUtilsTests.ts:17]: why are we removing these tests?

Comment (hellochar) [reply test/utils/windowUtilsTests.ts:17]: They tested behavior that was only possible in ie9 (a null `console` object)

File: src/utils/windowUtils.ts (hunk 1 above)
Comment (hellochar) [reply src/utils/windowUtils.ts:15]: 👍  https://github.com/palantir/plottable/issues/3314

**Review by leebyp — APPROVED** [review]

**Pull request merged**